packages/graphic-walker/src/components/pivotTable/index.tsx

```diff
@@ -238,7 +250,7 @@ const PivotTable: React.FC<PivotTableProps> = observer(function PivotTableCompon
                             enableCollapse={enableCollapse}
                         />
                     )}
-                    {metricTable && <MetricTable matrix={metricTable} meaInColumns={measInColumn} meaInRows={measInRow} />}
+                    {metricTable && <MetricTable matrix={metricTable} meaInColumns={measInColumn} meaInRows={measInRow} numberFormat={format} />}
```

Comment:
You can directly use `numberFormat={layout.format.numberFormat}` to pass the numberFormat, so that the table content will change immediately after modifying the numberFormat.